### DISCUSSION TITLE
Accelerate constraint preprocessing

### DISCUSSION THREAD

**Discussion by zoharl3** [yalmip/YALMIP#1022] [General discussion] [answered]
Any tips for that?

I have a hotspot at 

mi.lmi2sedumistruct>recursive_lp_fix, 12668 calls

Probably not--matlab and loops, and too many constraints--but I thought to ask.

**Comment by johanlofberg** [comment, marked as answer]
Vectorize the constraint definition. This only happens if you have used a for-loop creating many small LP constraints and the engine has to analyze and pack all these together

**Reply by zoharl3** [reply]
Guilty as charged, I'll need a more proper formulation.
